### PR TITLE
Release

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR bumps the version to 0.1.0.

# Checklist

- [X] I used `npm version <major|minor|patch>` to update `package.json`, inspecting the changelog to determine if the release was major, minor or patch.
- [X] The CHANGELOG has been updated to show version and release date - https://keepachangelog.com/en/1.0.0/.
- [X] The **only** commits in this PR are:
  - the CHANGELOG update.
  - the version update.
  - `@since` annotations.
- [X] I will make sure **not** to squash these commits, but **rebase** instead.
- [ ] Once this PR is merged, I will push the tag created by `npm version ...` (e.g. `git push origin vX.Y.Z`).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+The following sections document changes that have been released already:
+
+## 0.0.1 - 2020-02-10
```
Comment (Vinnl) [inline CHANGELOG.md:11]:
```suggestion
## 0.1.0 - 2020-02-10
```

**Review by Vinnl — APPROVED** [review]
Just the version nr in the changelog is wrong :)

File: CHANGELOG.md (hunk 1 above)
Comment (NSeydoux) [reply CHANGELOG.md]: Oops ^^

**Comment by NSeydoux** [comment]
Note for later: I think I'll keep the release commits in this order from now on (first `npm version`, and then changelog update), because fixes on the changelog are much more likely, but if they happen after the `npm version` commit the tag doesn't need to be updated.

**Pull request merged**